eng/pipelines/templates/steps/sync-repo-branch.yml

```diff
@@ -79,9 +79,10 @@ steps:
           if (!$TargetBranch) {
             $TargetBranch = $defaultBranch
           }
+          Write-Host ${{ target.value.Rebase }}
           if (-not '${{ target.value.Rebase }}') {
             git checkout -B target_branch source_branch
-            git push Target "target_branch:refs/heads/${TargetBranch}"
+            git push --force Target "target_branch:refs/heads/${TargetBranch}"
```

Comment:
Why are we force pushing? We should never have any scenario in our syncing that requires a force push.